lib/deimos/active_record_consume/mass_updater.rb

```diff
@@ -84,15 +87,60 @@ def import_associations(record_list)
         end
       end
 
+      # rubocop:disable Metrics/AbcSize
```

Comment:
Let's not disable this - if it's complaining, the function is too big. Let's break it up.